homeassistant/components/sabnzbd/__init__.py

```diff
@@ -50,18 +54,44 @@
 
 SIGNAL_SABNZBD_UPDATED = "sabnzbd_updated"
 
+
+@dataclass
+class SensorEntry(SensorEntityDescription):
+    """Metadata for sensor."""
+
```

Comment:
This is not adding anything and can be removed?

Reply:
Done